js/src/vercel.ts

```diff
@@ -202,6 +208,59 @@ function convertToDottedOrderFormat(
   );
 }
 
+function joinDotOrder(...segments: (string | undefined | null)[]): string {
+  return segments.filter(Boolean).join(".");
```

Comment:
Assuming this is equivalent to `s != null`

Reply:
Also catches empty segment string, which feels correct to me